tests/cephadm/test_daemon.py

```diff
@@ -26,16 +26,22 @@ def run(ceph_cluster, **kw):
           output, error   returned by the command.
     """
     config = kw.get("config")
-
-    # Manage Ceph using ceph-admin orchestration
-    command = config.pop("command")
-    log.info("Executing service %s service" % command)
-    daemon = Daemon(cluster=ceph_cluster, **config)
-
-    try:
-        method = fetch_method(daemon, command)
-        method(config)
-    finally:
-        # Get cluster state
-        get_cluster_state(daemon)
+    tests = config.get("steps") or [{"config": config}]
+
+    for test in tests:
+        # Manage Ceph daemon using ceph-admin daemon orchestration commands
+        _test_cfg = test["config"]
+        command = _test_cfg.pop("command")
+        log.info("Executing service %s service" % command)
+        daemon = Daemon(cluster=ceph_cluster, **_test_cfg)
+
+        try:
+            method = fetch_method(daemon, command)
+            method(_test_cfg)
+        except Exception as err:
+            log.error(err)
+            return 1
+        finally:
+            # Get cluster state
+            get_cluster_state(daemon)
```

Comment:
What exactly this function exactly validating ?

Reply:
It's just cluster state after operation.

Reply:
Do you think that would be sufficient ?

Reply:
I did not change this part of the code now. This PR is to support multi daemon operations for some of the RADOS tests.